### DISCUSSION TITLE
How to override the Markdown style of the editor?

### DISCUSSION THREAD

**Discussion by azuriteemporium** [outline/rich-markdown-editor#503] [Q&A]
Thank you a ton for making this project available to us all!

I was trying to override the markdown styling of the editor by wrapping the editor component in another div, for instance:

```
  import 'github-markdown-css';
 <Editor className='markdown-body' />
```

However, it results in the default markdown style now being mixed with the custom one 'markdown-body' Is there a away to achieve this?

**Comment by tommoor** [comment]
This is the nature of CSS, no? You can override but not replace.
